diff -b -u a/rsvg-cairo-draw.c b/rsvg-cairo-draw.c
--- a/rsvg-cairo-draw.c	2016-01-07 23:12:03.000000000 +0100
+++ b/rsvg-cairo-draw.c	2016-03-24 17:32:58.259054604 +0100
@@ -494,7 +494,7 @@
      * paths for the icon's shape.  We need to be able to compute the bounding
      * rectangle's extents, even when it has no fill nor stroke.
      */
-    {
+    if (state->fill != NULL) {
         RsvgBbox fb;
         rsvg_bbox_init (&fb, &state->affine);
         cairo_fill_extents (cr, &fb.rect.x, &fb.rect.y, &fb.rect.width, &fb.rect.height);
diff -b -u a/rsvg-convert.c b/rsvg-convert.c
--- a/rsvg-convert.c	2015-12-02 02:18:33.000000000 +0100
+++ b/rsvg-convert.c	2016-03-24 17:40:39.895065605 +0100
@@ -321,6 +321,11 @@
             if (!rsvg_handle_get_dimensions_sub (rsvg, &dimensions, export_lookup_id))
                 g_printerr ("Could not get dimensions for file %s\n", args[i]);

+            /* Offset the loss of precision of going from double to int.
+             * Without this fix, bottom right corner of bounding boxes is cut. */
+            dimensions.width += 2;
+            dimensions.height += 2;
+
             unscaled_width = dimensions.width;
             unscaled_height = dimensions.height;
